sms_utilities/requirements.txt

```diff
@@ -1 +1 @@
-smspdu==1.1
+smspdu==1.0
```

Comment:
why the downgrade? Or is this an artifact of your PR not being based on the current head release? @kkroo recently made changes to smspdu and sms_utilities to make them Py3 compatible. You probably should rebase against the current head to revert these.